angular/theme.scss

```diff
@@ -59,11 +59,11 @@ $material-theme: mat-light-theme($material-theme-primary, $material-theme-accent
                 secondary-text: map-get($pxb-black, 200),
                 hint-text: map-get($pxb-gray, 500),
                 base: map-get($pxb-darkBlack, 900),
-                divider: $dark-dividers,
+                divider: rgba(66, 78, 84, 0.12),
                 dividers: $dark-dividers,
-                disabled: $dark-disabled-text,
+                disabled: rgba(66, 78, 84, 0.3),
                 disabled-button: rgba(black, 0.26),
-                disabled-text: $dark-disabled-text,
+                disabled-text: rgba(66, 78, 84, 0.3),
```

Comment:
Would it make sense for us to override the values of `$dark-dividers` and `$dark-disabled-text` instead of changing to a hardcoded value?

Reply:
I don't think it matters much because we'll never use those variables directly; we'll reference those hard-coded values as map-get($foreground, divider), etc.